index.html

```diff
@@ -0,0 +1,140 @@
+            </path>
+          </g>
+        </svg>
+      </div>
+      <div class="top-nav-action">
+        <ul class="list-inline">
+          <li class="list-inline-item"><a class="btn d-none d-lg-block" href="#">Login</a><a class="d-lg-none" href=""><i class="bi bi-person-fill"></i></a></li>
+          <li class="list-inline-item"><a class="btn subscribe py-0 px-3 text-dark fw-bolder" href="#">Subscribe </a></li>
+        </ul>
+        <span class="genericon genericon-menu"><span></span></span>
+      </div>
+    </div>
+    <nav class="header-bottom d-none d-lg-block ">
+      <div class="mid-nav d-flex justify-content-between">
+        <ul class="list-unstyled d-flex flex-grow-1 pe-3 justify-content-between">
+          <li><a href="#">U.S.</a></li>
+          <li><a href="#">World</a></li>
+          <li><a href="#">Business</a></li>
+          <li><a href="#">Tech & Science</a></li>
+          <li><a href="#">Culture</a></li>
+          <li><a href="#">Newsgeek</a></li>
+          <li><a href="#">Sports</a></li>
+          <li><a href="#">Health</a></li>
+          <li><a href="#">The Debate</a></li>
+          <li><a href="#">Experts</a></li>
+          <li><a href="#">Vantage</a></li>
+        </ul>
+        <div class="mid-nav-search me-5">
+          <input type="text" placeholder="Search...">
+          <i class="bi bi-search"></i>
+        </div>
+      </div>
+      <div class="bottom-nav ps-4">
+        <ul class="list-unstyled d-flex flex-grow-1 pe-3">
+          <li><strong>TRENDING</strong></li>
+          <li><a class="" href="#">JOE BIDEN</a></li>
+          <li><a href="#">DONALD TRUMP</a></li>
+          <li><a href="#">INAUGURATION</a></li>
+          <li><a href="#">CORONAVIRUS</a></li>
+          <li><a href="#">STIMULUS PACKAGE</a></li>
+          <li><a href="#">VACCINE</a></li>
+          <li><a href="#">CAPITOL RIOTS</a></li>
+        </ul>
+      </div>
+    </nav>
```

Comment:
**In mobile and tablet screen sizes, the menu should hide, and show a hamburger button**

Reply:
It is hidden on medium screens, the Hamburger button is missing, and I am working on it!